modules/data/data-recorder/conf/recorder.debug.yaml

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
Could you change config files to protobuf, as we do in other modules?
As a python example:

Proto: https://github.com/ApolloAuto/apollo/blob/master/modules/hmi/proto/config.proto
Config file: https://github.com/ApolloAuto/apollo/blob/master/modules/hmi/conf/config.pb.txt
To load: https://github.com/ApolloAuto/apollo/blob/master/modules/tools/common/proto_utils.py#L22